components/SummaryComponents/RepoOverviewTable/ReposOverviewTable.tsx

```diff
@@ -1,56 +1,77 @@
 import Table from "@mui/material/Table";
-import TableCell from "@mui/material/TableCell";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
 import TableRow from "@mui/material/TableRow";
 import styles from "./ReposOverviewTable.module.css";
 import RedIcon from "../../../components/images/redIcon.svg";
 import YellowIcon from "../../../components/images/yellowIcon.svg";
 import greenIcon from "../../../components/images/greenIcon.svg";
 import Image from "next/image";
+import { TableBody } from "@mui/material";
+
+type RowProps = {
+  icon: any;
+  iconAlt?: string;
+  statusCount: number;
+  statusLabel: string;
+};
+
+enum StatusLabel {
+  RED = "need updating now",
```

Comment:
typo should be `needs`

Reply:
actually make it `needs updating urgently`

Reply:
I think it works because you can have multiple, i.e. '`58 need updating now`' is more grammatically correct than '`58 needs updating now`'. I thought the same thing at first and then realised ahaha.

Reply:
I see, is this better `need(s) updating urgently`, what do you think?